.circleci/config.yml

```diff
@@ -959,22 +915,24 @@ workflows:
           <<: *only-deployed-lower-environments
           requires:
             - wait-for-reindex
-      # - smoketests: # TODO 10007 Cleanup: Add back running readonly smoketests after 10007 has been deployed
-      #     <<: *only-deployed-lower-environments
-      #     requires:
-      #       - disable-reindex-cron
-      # - loadtests:
-      #     <<: *only-deployed-lower-environments
-      #     requires:
-      #       - disable-reindex-cron
-      # - smoketests-readonly: # TODO 10007 Cleanup: Add back running readonly smoketests after 10007 has been deployed to ustc/prod
-      #     <<: *only-deployed-lower-environments
-      #     requires:
-      #       - loadtests
-      - switch-colors:
+      - smoketests:
           <<: *only-deployed-lower-environments
           requires:
             - disable-reindex-cron
+      - loadtests:
+          <<: *only-deployed-lower-environments
+          requires:
+            - disable-reindex-cron
+      - smoketests-readonly:
+          <<: *only-deployed-lower-environments
+          requires:
+            - disable-reindex-cron
```

Comment:
one observation is that previously `smoketests-readonly` only happened after `smoketests` / `disable-reindex-cron` happened. I did observe one failure last week when they ran in parallel. Something to keep an eye on.